cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -1274,20 +1317,26 @@ def _ids_from_calibration_name(calibration_name: str) -> tuple[str, str, int]:
 def _quantum_processor_config_name_from_device_config_revision(
     project_id: str,
     processor_id: str,
-    config_name: str,
+    config_name: str | None = None,
     device_config_revision: DeviceConfigRevision | None = None,
 ) -> str:
     processor_resource_name = _processor_name_from_ids(project_id, processor_id)
     if isinstance(device_config_revision, Snapshot):
         return (
             f'{processor_resource_name}/'
-            f'configSnapshots/{device_config_revision.id}/'
-            f'configs/{config_name}'
+            f'configSnapshots/{device_config_revision.id}/configs'
+            f'/{config_name}'
```

Comment:
The slash will also be omitted since in this case the parent resource should name the snapshot/run:

`projects/<project_id>/processors/<processor_id>/configAutomationRuns/current`